src/SprykerEco/Yves/Adyen/Theme/default/prepayment.twig

```diff
@@ -0,0 +1,3 @@
+<div class="adyen-payment-form adyen-payment-sofort-form payment-form" id="adyenPrepayment">
```

Comment:
This is not sofort template.

Reply:
That was adde by our awesome frontend developer @alex-galych. I think he will create an issue and assign it to responsible person.